draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -880,7 +880,60 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 # IANA Considerations {#iana_considerations}
```

Comment:
We probably should keep some kind of TBD in here as a reminder for other things - like JWT claims - that'll need registration.

Reply:
```suggestion
# IANA Considerations {#iana_considerations}

TBD
```